steps/zip.sh

```diff
@@ -36,6 +36,12 @@ if [ -e "${zip}" ]; then
     exit
 fi
 
+cam_repo_target_dir="${TARGET}/github/self/${name}"
+
+if [ ! -d "${cam_repo_target_dir}" ]; then
+    git clone --depth 1 https://github.com/yegor256/cam.git "${cam_repo_target_dir}"
```

Comment:
@Raleksan I believe, it is reasonable to delete the `"${cam_repo_target_dir}/.git"` directory after the clone

Reply:
@yegor256 I know that I need to delete `.git`, therefore I locate it at `${TARGET}/github/self/${name}` to delete `.git` directory with all others git directories, using already existing command, or it is better to do it separately and explicitly?

Reply:
@Raleksan this location is wrong, I believe. Let's put it into `${TARGET}/cam-sources` instead

Reply:
@yegor256 I changed the location as you remarked and improved the test.

Can you please review it again?